src/main/java/seedu/souschef/logic/commands/AddCommand.java

```diff
@@ -36,6 +43,28 @@
     public static final String MESSAGE_SUCCESS = "New recipe added: %1$s";
     public static final String MESSAGE_DUPLICATE_RECIPE = "This recipe already exists in the address book";
 
+    public static final String MESSAGE_USAGE_HEALTHPLAN = COMMAND_WORD + ": Adds a healthplan. "
```

Comment:
I am also thinking of making additional class to sore these messages, which takes in context as a parameter. I will also fix it before Wednesday, so just leave it as it is for now!